### DISCUSSION TITLE
Please add Safari support

### DISCUSSION THREAD

**Discussion by vmanikes** [dailydotdev/daily#884] [Feature Requests]
Hey folks

Please add safari support, I am an avid user of safari and user of your platform. But it really sucks to switch to chrome to use your platform.

**Comment by idoshamun** [comment]
Hi @vmanikes,

Thanks for the suggestion. The are some technical issues with port our extension to Safari.
It's in our backlog and we hope to deal with it in the upcoming months.
In the meantime, feel free to use our [webapp](https://app.daily.dev) which is supported by all major browsers
